src/shared/HtmlReady.js

```diff
@@ -110,7 +110,8 @@ export default function(html, { mutate = true, hideImages = false } = {}) {
         };
     } catch (error) {
         // xmldom error is bad
-        throw new Error('HtmlReady: xmldom error');
+        console.log('rendering error', { error: error.message, html });
```

Comment:
Do a `JSON.encode` here so we are sure to get it on a single line in the logs

Reply:
smart; done